packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -251,6 +270,13 @@ export type ScreenShareButtonSelector = (state: CallClientState, props: CallingB
 // @public
 export const screenShareButtonSelector: ScreenShareButtonSelector;
 
+// @public
+export type StartCaptionsButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
+    isCaptionsFeatureActive: boolean;
+    currentCaptionLanguage: string;
```

Comment:
So the pop up window will be part of the button implementation?

Reply:
Will need to include current languages when starting captions